### ISSUE TITLE
CallDepth always starts at 2

### ISSUE THREAD

**Issue by kajdreef** [spideruci/blinky#14]
Below is a 'grep' of all the instruction execution events from an execution trace generated by blinky. It seems like there is an offset for the call depth by 1, because the call depth should start at 2 instead of 1. I assume.

```
$$$,1,1,0,2,C,3,$enter$,NA
$$$,2,1,5,3,856419764,5,$enter$,NA
$$$,3,1,5,3,856419764,6,$exit$
$$$,4,1,5,3,856419764,7,$enter$,NA
$$$,5,1,5,4,856419764,9,$enter$,NA
$$$,6,1,6,5,856419764,11,$enter$,NA
$$$,7,1,6,6,856419764,13,$enter$,NA
$$$,8,1,6,7,856419764,15,$enter$,NA
$$$,9,1,6,7,856419764,16,$exit$
$$$,10,1,7,2,C,4,$exit$
```

(Done with the latest version (41c3fd8) of blinky)

**Comment by VijayKrishna** [comment]
@kajdreef can you verify if this bug is resolved with commit id 34a0145?

**Comment by kajdreef** [comment]
Yes, upon entering the main the call depth is indeed 1 for my test programs! Thanks 👍
